client/state/posts/likes/reducer.js

```diff
@@ -19,9 +19,23 @@ import itemSchema from './schema';
 export const itemReducer = withSchemaValidation(
 	itemSchema,
 	( state = { likes: undefined, iLike: false, found: 0, lastUpdated: undefined }, action ) => {
+		/*
+		 * NOTE: This reducer includes some subtle, non-obvious behavior.
```

Comment:
Thanks for adding the comment! Just double checking it's up to date with any further changes you made? :)

Reply:
I can confirm this is accurate -- I added most of the content here quite late in the PR (thankfully!).